### ISSUE TITLE
Amplify can't find project when I do amplify codegen models

### ISSUE THREAD

**Issue by afern247** [aws-amplify/amplify-ios#1375]
### Describe the bug

I can do `Amplify push` and add apis etc, but when I do: `amplify codegen models` it says:

```
Successfully generated models. Generated models can be found in /Users/.../projectName/projectName/amplify/generated/models
Updating Xcode project:
🚫 xcodeProject: Target projectName not found
-- Recovery suggestion: Manually add Amplify files to your Xcode project.
```

Why it can't find it? Where is it defined so I can check if it's a name mismatch or it's an actual bug? 

When ask about modifying backend I selected yes.

### Amplify Categories

API, Auth, DataStore

### Dependency manager

Cocoapods

### Swift version

5.5

### CLI version

5.3.0

### Xcode version

13.0 beta 4

### iOS Version

15

**🟢 EDIT:**

I just discovered it's a naming issue, my target name is: `projectName (iOS)` and Amplify is looking for `projectName` how can I fix this?

This is extremely useful for api changes

**Comment by lawmicha** [comment]
> I just discovered it's a naming issue, my target name is: projectName (iOS) and Amplify is looking for projectName how can I fix this?

thanks for the update, I've seen issues related to different naming variations. How did you get to that state exactly? Was it when you created the App like File -> New -> Project -> App, and you set the Product Name to `projectName (iOS)` ?

**Comment by afern247** [comment]
So I created the project, initialized Amplify normally and it would work, I could do `amplify codegen models` the issue came after I renamed my app. Amplify were still looking for the old name, I also renamed the root folder where I had the files you know you have appName/appName/Views.. etc

I think there has to be a way to tell Amplify where to look for the project and for which project name to look for. I know amplify is in the root of the project and automagically knows it's inside of it, however on app name changes it gets lost.
